README.md

```diff
@@ -1,8 +1,22 @@
 # wp-cldr
 
-WordPress plugin to access localized territory and language names, currency names/symbols, and other localization info. Source is the [JSON distribution] (https://github.com/unicode-cldr/cldr-json) of the [Unicode Common Locale Data Repository (CLDR)] (http://cldr.unicode.org/).
+This plugin provides WordPress developers with easy access to localized country/region names, language names, currency names/symbols/usage, and other localization info from the [Unicode Common Locale Data Repository (CLDR)] (http://cldr.unicode.org/).
 
-This repository includes [locale display names] (https://github.com/unicode-cldr/cldr-localenames-modern) and [number formatting] (https://github.com/unicode-cldr/cldr-numbers-modern) for the locales currently [used by WordPress.com](https://github.com/Automattic/wp-cldr/blob/master/prune-cldr-files.php#L18) and [used by WordPress.org](https://github.com/Automattic/wp-cldr/blob/master/prune-cldr-files.php#L29), as well as the [core set of supplemental data] (https://github.com/unicode-cldr/cldr-core).
+With the plugin active, WordPress developers will have access across almost 100 WordPress locales to localized data items including:
```

Comment:
Not sure why, but this reads a bit odd to me -- maybe replace "almost 100" with "nearly 100"